css/my_portfolio.css

```diff
@@ -451,3 +451,448 @@ button:active {
+    width: 80%;
+  }
+
+  .my-work {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .card-work {
+    margin-top: 10%;
+  }
+
+  .cards {
+    margin-top: 10%;
+    display: flex;
+    margin-left: 5%;
+  }
+
+  .card-header {
+    display: flex;
+  }
+
+  .card-header-name {
+    color: #091e42;
+    font-family: 'Crete Round', sans-serif;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    margin-left: 10%;
+  }
+
+  .card-header-border {
+    width: 50%;
+    border: 1px solid #212121;
+    margin: 2.5%;
+  }
+
+  .card-text {
+    text-align: left;
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .story {
+    width: 33%;
+  }
+
+  .card-name {
+    margin-left: 0;
+  }
+
+  .skills {
+    width: 95%;
+  }
+
+  .skill1 {
+    margin: 4% 1% 4% 4%;
+    border: 1px solid #8993a4;
+    width: 4em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill2 {
+    margin: 4% 2%;
+    border: 1px solid #8993a4;
+    width: 4em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill3 {
+    margin: 4% 1% 4%;
+    border: 1px solid #8993a4;
+    width: 8em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill4 {
+    margin: 4% 1% 4% 1%;
+    border: 1px solid #8993a4;
+    width: 4em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .card-background-image-lg1 {
+    background-image: url("../assets/images/Img_Placeholder5.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+  }
+
+  .card-background-image-lg2 {
+    background-image: url("../assets/images/Img_Placeholder6.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+    left: 30%;
+    top: 58%;
+  }
+
+  .card-background-image-lg3 {
+    background-image: url("../assets/images/Img_Placeholder7.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+    left: 60%;
+    top: 58%;
+  }
+
+  .card-background-image-lg4 {
+    background-image: url("../assets/images/Img_Placeholder8.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+    top: 87.2%;
+  }
+
+  .card-background-image-lg5 {
+    background-image: url("../assets/images/Img_Placeholder6.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+    left: 30%;
+    top: 87.2%;
+  }
+
+  .card-background-image-lg6 {
+    background-image: url("../assets/images/Img_Placeholder7.png");
+    height: 27em;
+    width: 27.4%;
+    margin: 8% 4%;
+    display: grid;
+    left: 60%;
+    top: 87.2%;
+  }
+
+  .card-headline {
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    text-align: left;
+    margin-left: 6%;
+    padding-top: 40%;
+    font-family: "Crete Round", sans-serif;
+    width: 105%;
+  }
```

Comment:
- [ ] It would be nice to use the `font-weight: normal` for this CSS declaration. This would make the design align with that of Figma's template.

Reply:
Ok. 
Added. Thanks